monitoring/uss_qualifier/reports/report.py

```diff
@@ -123,3 +123,41 @@ class TestScenarioReport(ImplicitDict):
 
     execution_error: Optional[ErrorReport]
     """If there was an error while executing this test scenario, this field describes the error"""
+
+
+class TestSuiteActionReport(ImplicitDict):
+    test_suite: Optional["TestSuiteReport"]
+    """If this action was a test suite, this field will hold its report"""
+
+    test_scenario: Optional[TestScenarioReport]
+    """If this action was a test scenario, this field will hold its report"""
+
+    def successful(self) -> bool:
+        if "test_suite" in self and self.test_suite is not None:
```

Comment:
We may want to check here for invalid report which would contain test_suite and test_scenario keys so test_scenario failures don't go unnoticed in case of future wrong class usage..

Reply:
Done